client/src/components/AdminPage.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+
+class AdminPage extends Component {
```

Comment:
If you don’t have state or refs, prefer normal functions.[Airbnb Style Guide](https://github.com/airbnb/javascript/tree/master/react#class-vs-reactcreateclass-vs-stateless)